src/main/java/duck/Event.java

```diff
@@ -0,0 +1,62 @@
+package duck;
+import java.time.LocalDateTime;
```

Comment:
Should there be a new line separator between package and imports?

```suggestion
package duck;

import java.time.LocalDateTime;
```